wandb/sdk/launch/builder/kaniko_builder.py

```diff
@@ -162,15 +163,20 @@ def from_config(
         build_job_name = config.get("build-job-name", "wandb-launch-container-build")
         secret_name = config.get("secret-name", "")
         secret_key = config.get("secret-key", "")
-        image = config.get("kaniko-image", "gcr.io/kaniko-project/executor:v1.11.0")
+        kaniko_image = config.get(
+            "kaniko-image", "gcr.io/kaniko-project/executor:v1.11.0"
+        )
+        image_uri = config.get("destination")
```

Comment:
note: I don't love the name destination for the config key in the builder. Why not registry-uri?

Reply:
`destination` is also the name of the kaniko argument for specifying where you want the image pushed